Nerfs Malfunctioning drone on away sites

Alex6511 opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->
🆑 
tweak: the malfunctioning drone on away sites has had its damage adjusted
/🆑 

The drone is the only ranged retaliate mob, it seems really strong right now as a result. This should bring it more in-line with other away site ranged mobs (which are still very powerful).

babydoll reviewed (changes requested):
Change the subtype on hydro map, the base type aren't used for away maps.

afterthought2 reviewed (dismissed):
A rapid ranged retaliate mob will outgun any opposition by a factor of 3, at least in volume of fire. Seems too much.

Alex6511 commented:
This isn't in reference to the hydro map, I found these on ~~slavery base~~ some away map I don't know the name of, so it affects them all.

Alex6511 commented:
@sabiram 
Clarified, it's smugglers and the type that is there is the type I'm changing here.

Alex6511 committed:
Nerfs malf drone

remove unused code

Fixes snow planet ore maps

They were so poor they didn't pass sanity. Now sanity is more tweakable.

Also fixes lava planet's ultra rich ore map actually being super poor.
